frontend/packages/ceph-storage-plugin/src/components/modals/storage-pool-modal/storage-pool-modal.tsx

```diff
@@ -121,7 +124,22 @@ export const StoragePoolModal = withHandlePromise((props: StoragePoolModalProps)
     }
   }, [isSubmit, newPool, newPoolLoadError, newPoolLoaded, timer]);
 
-  const replicaDropdownItems = _.keys(OCS_DEVICE_REPLICA).map((replica) => {
+  const [storageCluster, storageClusterLoaded, storageClusterLoadError] = useK8sGet<
+    ListKind<StorageClusterKind>
+  >(OCSServiceModel, null, CEPH_STORAGE_NAMESPACE);
+
+  React.useEffect(() => {
+    setArbiterCluster(checkArbiterCluster(storageCluster?.items[0]));
+    if (isArbiterCluster) {
+      setReplicaSize('4');
+    }
+  }, [storageCluster, storageClusterLoaded, storageClusterLoadError, isArbiterCluster]);
```

Comment:
```suggestion
const storageClusterMemoized = useDeepCompareMemoize(storageCluster, true);
const isArbiterCluster = checkArbiterCluster(storageClusterMemoized);
```

Reply:
ack

Reply:
As we discussed, i am keeping this change as it is